### ISSUE TITLE
Previews do not recognize which books are affected by image updates

### ISSUE THREAD

**Issue by lcawl** [elastic/docs#1834]
If a PR affects on images (e.g. https://github.com/elastic/elasticsearch/pull/56359), the diff only lists the updated images. It does not include links to any of the books/pages that are affected.

Is this a known limitation with the diffs?
If it's not something we can fix, perhaps we can add a warning or recommendation that manual action is required to find and review the final changes?

**Comment by gtback** [comment]
Yep, if the image files are changed (but not renamed), then this won't result in any changes to the HTML files, so they won't show up in the diff. 

We could probably figure out all HTML pages that include the images that have changed and display that in addition on the diff page.

**Comment by lcawl** [comment]
If that's feasible, I think that would be great.
